app/assets/src/components/Samples.jsx

```diff
@@ -2180,4 +2305,31 @@ function AddUserModalMemberArea({ state, parent }) {
   );
 }
 
+function CohortButton({ label, action, enabled }) {
```

Comment:
Can you move this to a 'buttons' subfolder? We should start organizing any components that might be shared.

Reply:
sure